examples/ads.amp.html

```diff
@@ -1949,5 +1959,6 @@ <h2>Zucks</h2>
       data-frame-id="_bda46cf5ac">
   </amp-ad>
 
+
```

Comment:
nit: Extra new line. 😄

Reply:
sorry sir, i will remove those new lines and push those files